packages/loading/src/Circle.tsx

```diff
@@ -1,27 +1,34 @@
 import * as React from 'react';
 import * as shortid from 'shortid';
 import styled from 'styled-components';
+import { variables } from '@kata-kit/theme';
 
 export interface LoadingCircleProps {
   /** Loading icon size. */
   size?: number;
   /** Additional CSS classes to give to the icon. */
   className?: string;
+  /** Loading color */
+  color?: string;
 }
 
-const Root = styled('div')`
+const Root = styled('div')<LoadingCircleProps>`
   display: inline-block;
   vertical-align: middle;
 
   & svg path,
   & svg rect {
-    fill: #006fe6 /* $kata-blue */;
+    fill: ${props => props.color};
   }
 `;
 
-const Circle: React.FC<LoadingCircleProps> = ({ className, size = 40 }) => {
+const Circle: React.FC<LoadingCircleProps> = ({
+  className,
+  size = 40,
+  color
```

Comment:
```suggestion
  color = variables.colors.kataBlue
```

Set default props-nya dari atas sini saja.

Reply:
kemarin udah ku set default props nya di destructor nya mas, cuman kalau props di kosongi dia tetep jadi hitam 😄